### ISSUE TITLE
Error updating the images

### ISSUE THREAD

**Issue by paoloantinori** [alexbelgium/hassio-addons#197]
**Which addon?**

* Addon name : Jackett NAS
* Addon version : `0.20.492`

**Describe the bug**
Starting from a couple of version I'm getting an error to download the images. Initially I thought this could have just been the case of a bad push on the image servers, but I've notice the same issue with the most recent image I've noticed today.
The problem in my case is on Jackett and on  Qbittorrent, I've also Jellyfin installed.
From the nature of the error, I suspect the issue is with my system, but I don't really have an idea on what to check or try. 

**To Reproduce**

**System**

* Supervisor version: `supervisor-2021.12.2`
* Host system version: `Home Assistant OS 7.2`
```
22-01-31 18:09:40 INFO (SyncWorker_7) [supervisor.docker.interface] Updating image ghcr.io/alexbelgium/jackett-aarch64:0.20.399 to db21ed7f/aarch64-addon-jackett_nas:0.20.492
22-01-31 18:09:40 INFO (SyncWorker_7) [supervisor.docker.interface] Downloading docker image db21ed7f/aarch64-addon-jackett_nas with tag 0.20.492.
22-01-31 18:09:53 ERROR (SyncWorker_7) [supervisor.docker.interface] Can't install db21ed7f/aarch64-addon-jackett_nas:0.20.492: 404 Client Error for http+docker://localhost/v1.41/images/create?tag=0.20.492&fromImage=db21ed7f%2Faarch64-addon-jackett_nas: Not Found ("pull access denied for db21ed7f/aarch64-addon-jackett_nas, repository does not exist or may require 'docker login': denied: requested access to the resource is denied")```

**Comment by alexbelgium** [comment]
Hi, you were unlucky to install the addon during a 2 day period where I had switched the logic from local build to autobuild. I had to revert it which means the only way is to fully uninstall and reinstall the addon...

Here is the linked issue from the HA supervisor : https://github.com/home-assistant/supervisor/issues/3414

Here is the full description : https://github.com/alexbelgium/hassio-addons/issues/177

Bottom line the only solution due to this bug in HA is to fully uninstall and reinstall the addon. Hopefully the data will be maintained but I can't be sure. Or you wait for the supervisor bug to be solved but I don't know how long this will take as they seem very busy

**Comment by paoloantinori** [comment]
Hi, thank you for the explanation.

Uninstalling and reinstalling fixed the issue indeed. And **apparently** I haven't lost any config in the process!

thanks again

**Comment by alexbelgium** [comment]
Thanks for the feedback !
